src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA1839 | Performance | Info | PreferIsExactSpellingIsTrueForKnownApisAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1839)
```

Comment:
We need a new branch for post .NET5 analyzers. Master is targeting .NET5, so is not the right place for this PR.

Reply:
This comment is no longer valid. Master now targets .NET6